js/controller.js

```diff
@@ -1222,6 +1222,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     setClosedCaptionsLanguage: function(){
+      //if saved language not in available languages, set to first available language
+      if (this.state.closedCaptionOptions.language == null || !_.contains(this.state.closedCaptionOptions.availableLanguages.languages, this.state.closedCaptionOptions.language)) {
+        this.state.closedCaptionOptions.language = this.state.closedCaptionOptions.availableLanguages.languages[0];
```

Comment:
Is it always a sure thing that array this.state.closedCaptionOptions.availableLanguages.languages is not empty?  Don't you also want to check whether it is not empty?